services/shotgrid_common/utils.py

```diff
@@ -64,16 +64,17 @@ def _sg_to_ay_dict(sg_entity: dict, project_code_field=None) -> dict:
         folder_type = sg_entity["type"]
 
     ay_dict = {
-        "type": ay_entity_type, # In the EH there are either `folder_type` or `task_type`.
+        # In the EntityHub there are either `folder_type` or `task_type`
```

Comment:
this is not readable anymore if it's not a line comment. However, I think we shouldn't limit this sync to just these two types